hazelcast/src/main/java/com/hazelcast/map/impl/MapContainer.java

```diff
@@ -140,7 +145,8 @@ public MapContainer(final String name, final Config config, final MapServiceCont
                 .build();
         this.queryEntryFactory = new QueryEntryFactory(mapConfig.getCacheDeserializedValues(),
                 serializationService, extractors);
-        this.globalIndexes = shouldUseGlobalIndex() ? createIndexes(true, -1) : null;
+        this.globalIndexRegistry = shouldUseGlobalIndex()
+                ? createIndexRegistry(true, GLOBAL_INDEX_NOOP_PARTITION_ID) : null;
```

Comment:
Do we call a similar method for partitionedIndexRegistry somewhere?

Reply:
yes, we also call this for partitioned indexes while creating a new record-store, i'm not changing its implementation here, doing mainly 2 things:
- naming changes(class/variable/method/constant)
- moving partitioned-index registry from partition-container to map-container.